.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 sudo: false
 go:
-    - 1.6
+    - 1.7
```

Comment:
It seems like copyright headers need to be updated for all of the files in this commit with the current year.